app/views/layouts/mailer.html.slim

```diff
@@ -5,8 +5,17 @@ html
     = stylesheet_link_tag('mailer', media: 'all')
   body
     .header
-      = image_tag(attachments["brave_logo_horz_h30.png"]&.url, alt: "Brave Software", class: "logo", height: 30)
-      span.title= content_for(:title)
+      .brand
     .content
       = yield
     .footer
+      .bat
+        a href="https://basicattentiontoken.org/"
```

Comment:
minor: i kinda feel like these URLs should be configured, but it's probably okay